guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md

```diff
@@ -851,7 +852,7 @@ has been replaced with the following argument:
 * Improved catalog/search/advanced search pages response time: We have optimized the products load flow on storefront, which result in significant improvement of category, product, checkout and search result pages response time. There is minimum a five times improvement for page response time under high load.
 
 <!--- MC-16005-->
-* Client-side performance has been optimized by moving non-critical JavaScript code to the bottom of the page along with the corresponding deferred parsing and evaluation of this code. This means that users can see rendered pages faster.
+* Client-side performance has been optimized by moving non-critical JavaScript code to the bottom of the page along with the corresponding deferred parsing and evaluation of this code. This means that users can see rendered pages faster. To enable this performance enhancement, you must navigate to **Stores** > **Configuration** > **Developer** > **JavaScript Settings** and enable the **Move JS code to the bottom of the page** option.
```

Comment:
Check with Leslie and/or Lori to see if this new feature was documented. There's no docMC1 issue attached to  MC-16005